lnurl/core.py

```diff
@@ -26,14 +26,22 @@ def encode(url: str) -> Lnurl:
         raise InvalidUrl
 
 
-def get(url: str, *, response_class: Optional[Any] = None, verify: Union[str, bool] = True) -> LnurlResponseModel:
+def get(
+    url: str,
+    *,
+    response_class: Optional[Any] = None,
+    verify: Union[str, bool] = True,
+    raise_for_status: Optional[bool] = True
```

Comment:
What's the rationale for making this optional? Backwards compatibility? If so, why not `False`?